upgrade/upgrade-scripts/upgrade

```diff
@@ -176,12 +180,13 @@ function upgrade_not_in_place() {
 	"$IMAGE_PATH/upgrade-container" start "$CONTAINER" ||
 		die "failed to start upgrade '$CONTAINER'"
 
-	"$IMAGE_PATH/upgrade-container" upgrade "$CONTAINER" not-in-place ||
+	"$IMAGE_PATH/upgrade-container" upgrade "$CONTAINER" ||
 		die "Failed to upgrade do not-in-place upgrade on '$CONTAINER'"
 
 	if [[ "$DLPX_UPGRADE_SKIP_VERIFY" != "true" ]]; then
-		"$IMAGE_PATH/verify" -c "$CONTAINER" "$@" ||
-			die "'$IMAGE_PATH/verify' '$*' failed in '$CONTAINER'"
+		"$IMAGE_PATH/upgrade-container" run "$CONTAINER" \
+			"$IMAGE_PATH/verify-jar" "$@" ||
+			die "'$IMAGE_PATH/verify-jar' '$*' failed in '$CONTAINER'"
```

Comment:
I was thinking about this a little, and I wonder... rather than always *not* running `verify-stack` here, should we instead check to see if in-place upgrade is potentially allowed, and only skip `verify-stack` when it wouldn't be allowed?

Specifically, I'm wondering about the case where the user chose to do a "not-in-place" upgrade, when "in-place" would have been allowed.. in that case, I think it should be possible to run `verify-stack`, right?

Rather than always assuming that when "not-in-place" is used, it's because "in-place" wasn't allowed.. does this make sense?